### PR TITLE
ddl: update ddl add index progress timely

### PR DESCRIPTION
**Pull request by winkyao**
Before this PR, we update the rowCount in `admin show ddl` when we finished batch tasks, which may confuse the user. This PR update the rowCount immediately after the indices added.

### CODE REVIEW

**Comment by winkyao** [comment]
@shenli @zimulala @coocood @ciscoxll PTAL

File: ddl/reorg.go [hunk 1]
```diff
@@ -70,6 +70,14 @@ func (rc *reorgCtx) isReorgCanceled() bool {
 	return atomic.LoadInt32(&rc.notifyCancelReorgJob) == 1
 }
 
+func (rc *reorgCtx) setNextHandle(doneHandle int64) {
+	atomic.StoreInt64(&rc.doneHandle, doneHandle)
```
Comment (ngaut) [inline ddl/reorg.go]: Do we need to use CompareAndSwap ?

**Comment by coocood** [comment]
LGTM

**Comment by ciscoxll** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

File: ddl/reorg.go [hunk 2]
```diff
@@ -70,6 +70,14 @@ func (rc *reorgCtx) isReorgCanceled() bool {
...
+}
+
+func (rc *reorgCtx) increaseRowCount(count int64) {
+	atomic.AddInt64(&rc.rowCount, count)
+}
+
 func (rc *reorgCtx) setRowCountAndHandle(count, doneHandle int64) {
```
Comment (zimulala) [inline ddl/reorg.go]: Could we remove this function? We can use `setNextHandle` and `setNextHandle` instead of it.

Comment (winkyao) [reply ddl/reorg.go]: setRowCountAndHandle is still using.

**Pull request merged**